scripts/utils/getFilesRecursively.js

```diff
@@ -0,0 +1,19 @@
+const fs = require('fs');
+const path = require('path');
+
+const isFile = (filepath) => fs.statSync(filepath).isFile();
+const isDirectory = (filepath) => fs.statSync(filepath).isDirectory();
+const pathJoin = (base) => (filepath) => path.join(base, filepath);
+
+const getDirectories = (filepath) =>
+  fs.readdirSync(filepath).map(pathJoin(filepath)).filter(isDirectory);
+
+const getFiles = (filepath) =>
+  fs.readdirSync(filepath).map(pathJoin(filepath)).filter(isFile);
+
+const getFilesRecursively = (filepath) =>
+  getDirectories(filepath)
+    .flatMap(getFilesRecursively)
+    .reduce((acc, file) => [...acc, file], getFiles(filepath));
```

Comment:
this `reduce` is `concat` 🙂

```suggestion
  getFiles(filepath).concat(
    getDirectories(filepath).flatMap(getFilesRecursively)
  )
```

Reply:
ooh good find